data-reporting-service/person-service/src/test/resources/rawDataFiles/provider/ProviderReporting.json

```diff
@@ -197,14 +197,14 @@
     "zip": "30025",
     "county": "Gwinnett County",
     "county_code": "13135",
-    "country_code": "840",
+    "country": "840",
```

Comment:
country code is being surfaced as opposed to the code_short_desc_txt

Reply:
Fixed.